TYPO3.Neos/Classes/TYPO3/Neos/TypoScript/VariantMenuImplementation.php

```diff
@@ -66,6 +74,11 @@ protected function buildItems()
                 $nodeInDimensions = null;
             }
 
+            // determine labels for target dimensions of node
+            array_walk($targetDimensions, function (&$dimensionValue, $dimensionName, $allDimensionPresets) {
+                $dimensionValue = ['value' => $dimensionValue, 'label' => $allDimensionPresets[$dimensionName]['presets'][$dimensionValue]['label']];
+            }, $allDimensionPresets);
+
```

Comment:
Wait how does that even work, since targetDimensions doesn't contain the actual target values anymore? I mean how is it used as this change does not contain any usage.